Jenkinsfile

```diff
@@ -85,7 +86,7 @@ String gcpLabel(String s) {
 }
 
 def runIntegrationTest(String description, String kubeprodArgs, String ginkgoArgs, Closure clusterSetup, Closure dnsSetup) {
-    timeout(120) {
+    timeout(240) {
```

Comment:
just noticed this, can it really take 4 hours for the tests to complete. Is this a realistic number?

Reply:
Reverted to 120.